build.gradle

```diff
@@ -78,7 +78,7 @@ subprojects {
     avroVersion = '1.8.2'
     orcVersion = '1.4.2'
     parquetVersion = '1.10.0'
-    hiveVersion = '3.1.0'
+    hiveVersion = '1.2.1'
```

Comment:
Please see my comment regarding shading, should it be sufficient to shade and relocate hive libraries to avoid conflict with Spark?

Reply:
@rdblue OK, make sense. Note that for a project that uses both presto and iceberg it is quite confusing that presto upgrades to 3.x hive and iceberg downgrades to 1.x.